src/lib/components/CmsForm.svelte

```diff
@@ -185,82 +186,89 @@
-			</li>
-			<li>
-				To include pictures, add a paragraph with only <code class="font-mono"
-					>[Picture=slug]
-				</code>. You can also set the width, height and fit:
-				<code class="font-mono">[Picture=slug width=100 height=100 fit=cover]</code> or
-				<code class="font-mono">[Picture=slug width=100 height=100 fit=contain]</code>
-			</li>
-			<li>
-				To include challenges, add a paragraph with only <code class="font-mono"
-					>[Challenge=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your challenge
-			</li>
-			<li>
-				To include sliders, add a paragraph with only <code class="font-mono">[Slider=slug]</code>,
-				where
-				<code class="font-mono">slug</code> is the slug of your slider. You can specify the autoplay
-				duration in milliseconds like this:
-				<code class="font-mono">[Slider=slug?autoplay=3000]</code>
-			</li>
-			<li>
-				To include a specification widget, add a paragraph with only <code class="font-mono"
-					>[Specification=slug]</code
-				>, where <code class="font-mono">slug</code> is the slug of your specification.
-			</li>
-			<li>
-				To include a tag widget, add a paragraph with only <code class="font-mono">[Tag=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your tag. You can specify the display
-				option like this:
-				<code class="font-mono">[Tag=slug?display=var-1]</code>
-			</li>
-			<li>
-				To include a tagProducts widget, add a paragraph with only <code class="font-mono"
-					>[TagProducts=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your tag. You can specify the display
-				option like this:
-				<code class="font-mono">[TagProducts=slug?display=img-3]</code>
-			</li>
-			<li>
-				To include a form widget, add a paragraph with only <code class="font-mono"
-					>[Form=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your form.
-			</li>
-			<li>
-				To include a countdown widget, add a paragraph with only <code class="font-mono"
-					>[Countdown=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your countdown.
-			</li>
-			<li>
-				To include a specification widget, add a paragraph with only <code class="font-mono"
-					>[Specification=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your specification.
-			</li>
-			<li>
-				To include a gallery widget, add a paragraph with only <code class="font-mono"
-					>[Gallery=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your gallery. You can specify the display
-				option like this:
-				<code class="font-mono">[Gallery=slug?display=var-1]</code>
-			</li>
-		</ul>
+		<label class="checkbox-label my-2">
+			<input type="checkbox" name="showTips" bind:checked={showTips} class="form-checkbox" />
+			Show tips
+		</label>
+		{#if showTips}
+			<ul class="text-gray-700 my-3 list-disc ml-4">
```

Comment:
you can do it in CSS-only:

```suggestion
		<label class="checkbox-label my-2 peer">
			<input type="checkbox" name="showTips" bind:checked={showTips} class="form-checkbox" />
			Show tips
		</label>
		<ul class="text-gray-700 my-3 list-disc ml-4 hidden peer-checked:block">
```

With no need for an extra var (and it works without JS in the client)